index.php

```diff
@@ -2,11 +2,36 @@
 require_once './functions.php';
 $db = new PDO('mysql:host=db; dbname=kantopokemon', 'root', 'password');
 $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
-$query = $db->prepare('SELECT * FROM `kanto`');
+$query = $db->prepare('SELECT * FROM `kanto pokemon plural`');
 $query->execute();
 $allPokemon = $query->fetchAll();
+?>
 
-echo "<h1>" . "Pokemon of the Kanto Region" . "</h1>";
+<html>
+<head>
+    <link rel="stylesheet" type="text/css" href="stylesheet.css">
+    <title>My Database</title>
+</head>
+<body>
+<header>
+    <section>
+        <div class="main-heading">
```

Comment:
you dont need a `div` here or a `class` - select using `h1` and `header` (as there should only be one of each on the page)